airflow/models/baseoperator.py

```diff
@@ -235,29 +234,26 @@ class derived from this one results in the creation of a task object,
     """
 
     # For derived classes to define which fields will get jinjaified
-    template_fields = []  # type: Iterable[str]
+    template_fields: Iterable[str] = []
     # Defines which files extensions to look for in the templated fields
-    template_ext = []  # type: Iterable[str]
+    template_ext: Iterable[str] = []
     # Defines the color in the UI
     ui_color = '#fff'  # type: str
     ui_fgcolor = '#000'  # type: str
 
     pool = ""  # type: str
 
     # base list which includes all the attrs that don't need deep copy.
-    _base_operator_shallow_copy_attrs = ('user_defined_macros',
-                                         'user_defined_filters',
-                                         'params',
-                                         '_log',)  # type: Iterable[str]
+    BASE_OPERATOR_SHALLOW_COPY_ATTRS: Tuple[str, ...] = \
+        ('user_defined_macros', 'user_defined_filters', 'params', '_log',)
 
     # each operator should override this class attr for shallow copy attrs.
-    shallow_copy_attrs = ()  # type: Iterable[str]
+    shallow_copy_attrs: Tuple[str, ...] = ()
 
     # Defines the operator level extra links
-    operator_extra_links = ()  # type: Iterable[BaseOperatorLink]
+    operator_extra_links: Iterable['BaseOperatorLink'] = ()
 
-    # Set at end of file
-    _serialized_fields = frozenset()  # type: FrozenSet[str]
+    _serialized_fields: Optional[FrozenSet[str]] = None
```

Comment:
Maybe keep the comment so that Readers know where to look for ?
```suggestion
     # Set at end of file
    _serialized_fields: Optional[FrozenSet[str]] = None
```

Reply:
OK. I will keep comment and explain it's lazily set in get_serialized_fields() method.

Reply:
Added.